src/cljc/harja/domain/yllapitokohde.cljc

```diff
@@ -1307,3 +1307,27 @@ yllapitoluokkanimi->numero
                 (= (select-keys kohdeosa tr-domain/vali-avaimet)
                    (select-keys rivi tr-domain/vali-avaimet)))
               kohdeosat))))
+
+(defn laske-sarakkeen-summa [sarake kohderivit]
+  (reduce + 0 (keep
+                (fn [rivi] (sarake rivi))
+                kohderivit)))
+
+(defn kohteiden-summarivi
+  "Laskee ylläpitokohteen kustannusriveistä summarivin."
+  [kohteet]
+  (let [sopimuksen-mukaiset-tyot-yhteensa (laske-sarakkeen-summa :sopimuksen-mukaiset-tyot kohteet)
```

Comment:
Ihan turhaa viilaamista, mutta voitetaanko me jotain, kun näin hyvin nimetyt mäpin avaimet tallennetaan letissä erikseen? Voisi siis kutsua alla olevassa mapissa suoraan.

Reply:
Totta, sama poistaa turha lettaus